ccstake/README.md

```diff
@@ -0,0 +1,207 @@
+# CodeChain Stakeholder CLI: ccstake
+
+A CLI tools for CodeChain stakeholders, and validators.
+
+## Features
+
+- CCS: [show](#show), [transfer](#transfer), [delegate](#delegate), [revoke](#revoke), [redelegate](#revoke), [batch-delegate](#batch-delegate)
+- Governance: [sign](#sign), [change-params](#change-params)
+- Validator: [validators](#validators), [self-nominate](#validators)
+
+## Usages
+
+You can see the actual usages with more contexts here.
+* [CodeChain User Guide: Staking](https://codechain.readthedocs.io/en/latest/user-guide/staking/)
+* [CodeChain User Guide: Running a Validator Node](https://codechain.readthedocs.io/en/latest/user-guide/running-a-validator-node/)
+
+### Installation
+
+```bash
+yarn global add codechain-stakeholder-cli
+```
+
+You also need cckey to manage the local keystore database: https://github.com/codechain-io/codechain-keystore-cli
+
+If you successfully installed the cli, you can see the overview with
+```bash
+$> ccstake
+ccstake <command>
+
+Commands:
+  ccstake show [account]                    Show staking status of an account
+  ccstake transfer                          Transfer CCS to an account
+  ccstake delegate                          Delegate CCS to an account
+  ccstake batch-delegate                    Batch manage delegations through
+  <distribution-file>                       distribution file
+  ccstake revoke                            Revoke delegation to an account
+  ccstake self-nominate                     Self nominate as a candidate
+  ccstake validators                        Show validators
+  ccstake sign                              Sign a message
+  ccstake change-params                     Change CodeChain network parameter
+
+Options:
+  --version     Show version number                                    [boolean]
+  --keys-path   The path to storing the keys [string] [default: "./keystore.db"]
+  --rpc-server  The RPC server URL
+                                 [string] [default: "https://rpc.codechain.io/"]
+  --help        Show help                                              [boolean]
+
+Not enough non-option arguments: got 0, need at least 1
+```
+
+### Managing CCS
+
+#### show
+
+To see overview of CCS distribution, you can use `show` command.
+
+```bash
+$> ccstake show
+```
+
+You can see specific account's CCS state by passing an address to the `show` command.
```

Comment:
You can see a specific account's CCS state by passing an address to the `show` command.